src/servers/Cargo.toml

```diff
@@ -30,7 +30,7 @@ metrics = "0.20"
 num_cpus = "1.13"
 once_cell = "1.16"
 openmetrics-parser = "0.4"
-opensrv-mysql = "0.2"
+opensrv-mysql = { git = "https://github.com/SSebo/opensrv.git" }
```

Comment:
Depending on my own repo for now, will replace to `opensrv-mysql` when this PR merged https://github.com/datafuselabs/opensrv/pull/34